src/html/comrak_adapters.rs

```diff
@@ -85,11 +85,11 @@ impl HighlightAdapter {
     write!(output, "</code>")?;
     write!(
       output,
-      r#"<button class="context_button" data-copy="{}">{}</button>"#,
+      r#"<button class="context_button" data-copy="{}">{}{}</button>"#,
       html_escape::encode_double_quoted_attribute(source),
-      include_str!("./templates/icons/copy.svg")
-    )?;
-    write!(output, "<code>")
+      include_str!("./templates/icons/copy.svg"),
+      include_str!("./templates/icons/check.svg")
+    )
```

Comment:
missing opening `code` tag as there was before

Reply:
the removal was intentional, as it created an empty `code` block.
should i reintroduce it?

Reply:
fair enough! reverted that change, thanks for the hint!